ui.R

```diff
@@ -0,0 +1,20 @@
+library(shinysky)
+library(shiny)
+library(maps)
+library(mapproj)
+
+my_autocomplete_list <- c("Web Developer", "Software Developer", "Finance", "Biologist",
+                          "Chemist", "Physist", "Doctor", "Engineer", "Manager")
+
+# Define UI for application that draws a histogram
+shinyUI(
+  navbarPage(
+    "Student Loan Data",
+    source('introduction.r', local=TRUE)$value,
```

Comment:
the `$value` is what removed the TRUE or FALSE value from the bottom of the page for anyone who was interested.